internal/js/modules/k6/browser/common/selectors.go

```diff
@@ -28,6 +28,12 @@ var reQueryEngine *regexp.Regexp = regexp.MustCompile(`^[a-zA-Z_0-9-+:*]+$`)
 // Matches start of XPath query.
 var reXPathSelector *regexp.Regexp = regexp.MustCompile(`^\(*//`)
 
+// Matches the text selectors of the form text=<value>
+var reTextSelector *regexp.Regexp = regexp.MustCompile(`^\s*text\s*=\s*(.*)$`)
+
+// ErrEmptySelector is returned when a selector string is empty.
+var ErrEmptySelector = errors.New("provided selector is empty")
```

Comment:
```suggestion
var errEmptySelector = errors.New("provided selector is empty")
```
It doesn't seem it is required to have it exported. Btw, do we really need to have it as a global variable? It seems we use it only one time, we could keep the allocation local to the function, then use `ErrorContains` in tests to match the most important part of the error.